From df04775feb13d9eabe595c34bc6c33ed3c8ae9d1 Mon Sep 17 00:00:00 2001
From: Ky-Anh Huynh <kyanh@theslinux.org>
Date: Thu, 11 Apr 2013 13:43:38 +0000
Subject: [PATCH] Add new module :xtelex

This module is a big table of predefined words for Telex users.
Pls check details at https://github.com/icy/myquartz-scim2uim
---
 uim-vi/src/uim-1.8.5/pixmaps/Makefile.am | 4 ++--
 uim-vi/src/uim-1.8.5/scm/Makefile.am     | 3 ++-
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/uim-vi/src/uim-1.8.5/pixmaps/Makefile.am b/uim-vi/src/uim-1.8.5/pixmaps/Makefile.am
index 24b4489..46a4324 100644
--- a/uim-vi/src/uim-1.8.5/pixmaps/Makefile.am
+++ b/uim-vi/src/uim-1.8.5/pixmaps/Makefile.am
@@ -25,12 +25,12 @@ SVGS += direct_input_dark_background.svg \
 SVGS += im_switcher.svg uim-dict.svg
 
 # input method substitutions
-IM_SUBST_SVGS = viqr.svg ipa-x-sampa.svg scim.svg
+IM_SUBST_SVGS = xtelex.svg viqr.svg ipa-x-sampa.svg scim.svg
 
 GENERATED_SVGS = $(IM_SUBST_SVGS)
 SVGS += im_subst.svg $(GENERATED_SVGS)
 
-IM_SUBST_PNGS = viqr.png ipa-x-sampa.png scim.png
+IM_SUBST_PNGS = xtelex.png viqr.png ipa-x-sampa.png scim.png
 
 SVG_PNGS = unknown.png \
            direct.png \
diff --git a/uim-vi/src/uim-1.8.5/scm/Makefile.am b/uim-vi/src/uim-1.8.5/scm/Makefile.am
index 331b219..636691f 100644
--- a/uim-vi/src/uim-1.8.5/scm/Makefile.am
+++ b/uim-vi/src/uim-1.8.5/scm/Makefile.am
@@ -18,6 +18,7 @@ SCM_FILES = plugin.scm im.scm im-custom.scm lazy-load.scm init.scm \
  generic.scm generic-custom.scm generic-key-custom.scm \
  generic-predict.scm predict-custom.scm \
  viqr.scm \
+ xtelex.scm \
  ipa-x-sampa.scm \
  m17nlib.scm m17nlib-custom.scm \
  uim-module-manager.scm \
@@ -44,7 +45,7 @@ EXTRA_DIST = $(OTHER_SCM_FILES)
 
 ETAGS_ARGS=$(SCM_FILES) $(GENERATED_SCM_FILES)
 
-module_names = "m17nlib" "viqr" "ipa-x-sampa"
+module_names = "m17nlib" "viqr" "xtelex" "ipa-x-sampa"
 
 UIM_MODULE_MANAGER = $(top_builddir)/uim/uim-module-manager
 UIM_MODULE_MANAGER_ENV = \
-- 
1.8.3

